app/models/manageiq/providers/ansible_tower/automation_manager/credential.rb

```diff
@@ -1,3 +1,7 @@
 class ManageIQ::Providers::AnsibleTower::AutomationManager::Credential < ManageIQ::Providers::ExternalAutomationManager::Authentication
+  alias_attribute :manager_id, :resource_id
+  alias_attribute :manager, :resource
```

Comment:
ok. maybe add a comment above, that `TowerApi` expects the object connected to its manager via `.manager` and `Credential` is connected via `resource`.
I dont know of a better approach